doc/readthedocs/developer_cluster.rst

```diff
@@ -7,7 +7,7 @@ In general there are several ways to get the |FOURC| source code to a cluster. I
 
 - Limitations due to firewall settings may emerge.
 - In case you want to commit changes in your repository on a cluster,
-  remember to set up |FOURC| as described in the |FOURC| ``README.md`` or in the :ref:`Setup guide <SetupGuideto4C>`.
+  remember to set up |FOURC| as described in the :ref:`Installation guide <installation>`.
```

Comment:
At second thought, one could argue that "installation" is not an optimal word here, since one usually does not `make install` 4C, but just `make` (build) it. On the other hand, "installation guide" conveys the proper meaning of "getting everything up and running".

I am fine with the current wording. Just wanted to raise awareness.